cellpack/autopack/utils.py

```diff
@@ -257,3 +257,26 @@ def get_value_from_distribution(distribution_options, return_int=False):
         value = int(numpy.rint(value))
 
     return value
+
+
+def get_seed_list(packing_config_data, recipe_data):
+    # Returns a list of seeds to use for packing
+    if packing_config_data["randomness_seed"] is not None:
+        seed_list = packing_config_data["randomness_seed"]
+    elif recipe_data.get("randomness_seed") is not None:
+        seed_list = recipe_data["randomness_seed"]
+    else:
+        seed_list = None
```

Comment:
Nit: should we implement an early return here for `seed_list` when it's `None`? So that we don't need to check it again in L274?